### ISSUE TITLE
Feature request: alphabetical sorting button and group by symbol checkbox 

### ISSUE THREAD

**Issue by fthebaud** [patrys/vscode-code-outline#148]
I really like the extension. I think it would be even nicer if it was easy to switch between the following sorting mode:
- no sorting (items appear in the order they are in the file)
- alphabetical sorting (ascending/descending)

Also it would be nice to be able to chose to group by symbols or not. (symbols groups would appear in the order defined in the configuration. Items in each symbol group would be sorted according to the current sort setting)

In the UI, this could take the form of sorting button:
**\/** : alphabetical sorting, ascending
**/\** : alphabetical order, descending
 **-** : no sorting
and of a "group by symbols" checkbox.

**Comment by fthebaud** [comment]
Well, looks like vscode 1.25.0 is bringing this sort by name feature...

**Comment by patrys** [comment]
I'm planning to deprecate this extension in the coming days once everyone gets a chance to upgrade to the latest VS Code so it's unlikely for me to implement any additional features. It does not make sense to compete with the built in outline.

**Comment by fthebaud** [comment]
I agree... Well thanks for maintaining this extension so far! It was pretty useful.
